app/models/user.rb

```diff
@@ -0,0 +1,8 @@
+class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable, :confirmable
+  has_many :recipes, dependent: :destroy
+  has_many :foods, dependent: :destroy
+end
```

Comment:
- [ ] At the moment, when a user signs up they are not redirected to the home page and there is no information telling them that they have registered successfully and need to confirm their email. Please kindly ensure this is fixed to also improve the user experience